### DISCUSSION TITLE
Invalid configuration parameter: CHANGELOG_SKIP_EMPTY_CHANGES

### DISCUSSION THREAD

**Discussion by ecurb91** [netbox-community/netbox#14856] [Help Wanted!]
Receiving the Invalid configuration parameter: CHANGELOG_SKIP_EMPTY_CHANGES error when trying to save anything within Netbox.  Have checked documentation and cannot see where this value is set. 

Any help would be appreciated. 
![Screenshot from 2024-01-19 08-21-55](https://github.com/netbox-community/netbox/assets/136307080/cc1f8170-d518-484a-9c80-772b1188f47e)

**Comment by candlerb** [comment]
You are reading the documentation for the current release of Netbox, and the setting `CHANGELOG_SKIP_EMPTY_CHANGES` was only introduced in v3.7.0: see "Enhancements" in the [release notes](https://github.com/netbox-community/netbox/releases/tag/v3.7.0).

Note also that your "v3.6.7-dev" is some unreleased version of Netbox, I guess somewhere from the "develop" branch between 3.6.6 and 3.6.7. Upgrades from this version to any later version are not supported and are completely at your own risk (i.e. you may end up with tables in invalid states). You should checkout the "master" branch instead, and for safety delete and recreate your database.

**Reply by ecurb91** [reply]
How do I ensure I am installing the latest version? when I clone the master branch or download a zip of it I always seem to install this dev version?

**Reply by candlerb** [reply]
I guess you have multiple versions of Netbox installed (in different directories) and the one that's running is not the one you think you are.
